juniper/src/integrations/chrono.rs

```diff
@@ -157,17 +150,16 @@ mod test {
 
     #[test]
     fn naivedate_from_input_value() {
-        naivedate_test("1996-12-19T16:39:57-08:00", 1996, 12, 19);
+        naivedate_test("1996-12-19", 1996, 12, 19);
     }
 
-    #[test]
-    fn naivedate_from_input_value_with_fractional_seconds() {
-        naivedate_test("1996-12-19T16:39:57.005-08:00", 1996, 12, 19);
-    }
+    fn naivedate_serialization() {
+        let date = NaiveDate::from_ymd(2000, 1, 2);
+        // let value = date.serialize().unwrap();
```

Comment:
@theduke can you tell me what needs to be called in order to serialize a `NaiveDate`. So convert this to an `InputValue` using the `resolve` function. Thanks

Reply:
Something like https://github.com/graphql-rust/juniper/blob/master/juniper/src/integrations/url.rs#L23 ?